test/index.test.js

```diff
@@ -1,3 +1,41 @@
-test('Initial test', () => {
-  expect(1).toBe(1);
+const request = require('supertest');
+const connection = require('../server/database/config/connection');
+const dbBuild = require('../server/database/config/build');
+
+const app = require('../server/app');
+
+beforeAll(() => {
```

Comment:
You don't need ``` return ``` here 😉
```suggestion
beforeAll(() => dbBuild());
```